src/index.html

```diff
@@ -9,6 +9,83 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+         src="images/logo.png"
+         alt="moyo-logo"
+         class="logo-moyo"
+         >
```

Comment:
Pay attention on indents
```suggestion
        <img
         src="images/logo.png"
         alt="moyo-logo"
         class="logo-moyo"
       >
```